source/nodejs/adaptivecards-fabric/src/Components/Inputs/InputTimeFabric.tsx

```diff
@@ -17,8 +17,8 @@ export class InputTimeFabric extends Shared.ReactInputElement {
         this.id = AC.getStringValue(json.id);
         this.value = json.value;
         this.placeholder = AC.getStringValue(json.placeholder);
-        this.min = AC.getStringValue(json.min);
-        this.max = AC.getStringValue(json.max);
+        this.min = json.min ? AC.getStringValue(json.min) : undefined;
```

Comment:
getStringValue's implementation is
```typescript
export function getStringValue(obj: any, defaultValue: string = undefined): string {
    return obj ? obj.toString() : defaultValue;
}
```
So you shouldn't have to do the test yourself.

Reply:
got it, will update